src/main/java/seedu/address/logic/Messages.java

```diff
@@ -19,6 +19,8 @@ public class Messages {
     public static final String MESSAGE_DUPLICATE_FIELDS =
                 "Multiple values specified for the following single-valued field(s): ";
     public static final String MESSAGE_INPUT_LENGTH_EXCEEDED = "The input length cannot exceed 50 characters";
+    public static final String MESSAGE_TOO_MANY_INDEXES = "The number of target guests cannot exceed 10.";
```

Comment:
Should string formatting be used here, so that this message can be extended to other commands that can support more indexes?